adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/WEB-INF/views/responsive/pages/checkout/multi/checkoutSummaryPage.jsp

```diff
@@ -32,7 +32,7 @@
                     </div>
                 </div>
                 <div class="place-order-form hidden-xs">
-                    <form:form action="${placeOrderUrl}" id="placeOrderForm1" commandName="placeOrderForm">
+                    <form:form action="${placeOrderUrl}" id="placeOrderForm1" modelAttribute="splaceOrderForm">
```

Comment:
just double checking this works as well for older 6.x versions ?

Reply:
Yes, I changed it locally and ran the server (I'm using 6.7), everything worked fine.